spark/src/main/scala/org/apache/spark/sql/comet/operators.scala

```diff
@@ -471,6 +488,9 @@ case class CometLocalLimitExec(
     child: SparkPlan,
     override val serializedPlanOpt: SerializedPlan)
     extends CometUnaryExec {
+
+  override def outputPartitioning: Partitioning = child.outputPartitioning
```

Comment:
Would it make sense to add this `outputPartitioning` to `CometUnaryExec` as the default?